pyuvdata/tests/test_uvbeam.py

```diff
@@ -1371,3 +1371,20 @@ def test_get_beam_functions():
     az_za_beam.data_normalization = 'peak'
     nt.assert_raises(ValueError, az_za_beam.get_beam_area)
     nt.assert_raises(ValueError, az_za_beam.get_beam_sq_area)
+
+    # check cross polarized beam (xy and yx)
+    efield_beam = UVBeam()
+    efield_beam.read_cst_beam(cst_files[0], beam_type='e-field', frequency=150e6,
+                             telescope_name='TEST', feed_name='bob',
+                             feed_version='0.1',
+                             model_name='E-field pattern - Rigging height 4.9m',
+                             model_version='1.0')
+
+    efield_beam.efield_to_power()
+    power_beam = copy.deepcopy(efield_beam)
+    power_beam.az_za_to_healpix()
+    power_beam.peak_normalize()
+    XY_area = power_beam.get_beam_sq_area("XY")
+    YY_area = power_beam.get_beam_sq_area("YX") 
```

Comment:
I think this should read "YX_area"

Also, there should probably be an assert in this test. Maybe at least test that the XY & YX areas are equal? Also, I think that they should be less than the XX or YY area, right?